doc/rst/language/spec/domains.rst

```diff
@@ -747,48 +734,45 @@ Domain Striding
 
 The ``by`` operator can be applied to a rectangular domain value in
 order to create a strided rectangular domain value. The right-hand
-operand to the ``by`` operator can either be an integral value or an
+operand to the ``by`` operator is the stride value,
+which can be either an integral value or an
 integral tuple whose size matches the domain’s rank.
 
-
-
 .. code-block:: syntax
 
    domain-striding-expression:
      domain-expression 'by' expression
 
-The type of the resulting domain is the same as the original domain but
-with ``stridable`` set to true. In the case of an integer stride value,
-the value of the resulting domain is computed by applying the integer
-value to each range in the value using the ``by`` operator. In the case
-of a tuple stride value, the resulting domain’s value is computed by
-applying each tuple component to the corresponding range using the
-``by`` operator.
+The type of the resulting domain is the same as the original domain,
+with the ``strides`` parameter adjusted to accept all the strides
+that are accepted by the ``strides`` parameters of the resulting
+domain's ranges.
```

Comment:
If you make a change like the one I suggested above, you'll want to replicate that language here.